main/monmon.rb

```diff
@@ -1,6 +1,7 @@
 require 'csv'
 
 balance = Hash.new(0)
+support_currency = ['BYN', 'USD', 'RUR', 'EUR']
```

Comment:
It's not supposed to be changed during the program run. Hence, it should be a constant.
A constant always has all capital letters.
Read about constants in Ruby https://www.rubyguides.com/2017/07/ruby-constants/

Also this concept applies for almost **all** other languages.

Reply:
there is a shortcut in ruby to create an array of strings: `c = %w[BUN USD RUR EUR]`

Reply:
Ok. I will fix